beacon_node/network/src/sync/range_sync/range.rs

```diff
@@ -75,24 +76,24 @@ pub struct RangeSync<T: BeaconChainTypes, C = BeaconChain<T>> {
     chains: ChainCollection<T, C>,
     /// Chains that have failed and are stored to prevent being retried.
     failed_chains: LRUTimeCache<Hash256>,
-    /// The syncing logger.
-    log: slog::Logger,
 }
 
 impl<T: BeaconChainTypes, C> RangeSync<T, C>
 where
     C: BlockStorage + ToStatusMessage,
     T: BeaconChainTypes,
 {
-    pub fn new(beacon_chain: Arc<C>, log: slog::Logger) -> Self {
+    pub fn new(beacon_chain: Arc<C>) -> Self {
+        let span = span!(Level::INFO, "service = range_sync");
```

Comment:
Had to remove  [this](https://github.com/sigp/lighthouse/pull/6339/commits/6cc4f18c7ad225c5d15a47a2c0b151e1bc94f3fc#diff-3d6c8081703730f63787240a11f38fdde1d721397245cf316207b108f1df3defL299), so added the span here to keep the additional context it would’ve provided.
Did the same treatment in other places where `slog::o!` was being used